spec/graphql/list_type_spec.rb

```diff
@@ -8,6 +8,10 @@
     assert_equal([1.0, 2.0, 3.0].inspect, float_list.coerce_isolated_input([1, 2, 3]).inspect)
   end
 
+  it "converts items that are not lists into lists" do
+    assert_equal([1.0].inspect, float_list.coerce_isolated_input(1.0).inspect)
```

Comment:
I imagine it works without it; I was just following the pattern of the other tests in this file, which also compare the results of `inspect`. I'll switch remove those calls, though.